packages/venia-ui/lib/components/WishlistPage/wishlist.css

```diff
@@ -19,6 +19,10 @@
     grid-auto-flow: column;
 }
 
+.emptyListText {
+    text-align: center;
```

Comment:
Minor UX observation; mocks appear to have a minimum height that isn't reflected here. Giving the grid container a `min-height` is preferable here, but a simple `line-height` here could accomplish the same thing. If you want to attempt `min-height` solution, you'll need to give the grid alignment instructions, since it appears to want to stretch. If this has been approved by UX, no big deal.

Reply:
My preference is to let the grid align the content, but this is fine 👍